en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -433,15 +442,18 @@ use SilverStripe\ORM\DataObject;
 
 class MyDataObject extends DataObject
 {
+    // ...
     private static $searchable_fields = [
         'Name' => 'PartialMatchFilter',
-        'ProductCode' => NumericField::class,
+        'ProductCode' => [
+            'field' => NumericField::class,
+        ],
```

Comment:
I don't know if this worked at some stage and broke, but it definitely doesn't work right now and shouldn't be documented as a thing that works.